src/main/resource/config.properties

```diff
@@ -6,17 +6,17 @@ geoserver.dbport=5432
 geoserver.dbhost=localhost
 geoserver.dbname=ibp
 geoserver.dbuser=postgres
-geoserver.web.username=nakshauser
-geoserver.web.password=1565707817
+geoserver.web.username=admin
+geoserver.web.password=geoserver
```

Comment:
@jadhavam1994 please do not commit unnecessary credentials in properties file remove this